tests/integration/example/integration.sh

```diff
@@ -26,35 +26,47 @@ function process_result() {
     fi
 }
 
-# Build mixer binary
-bazel build //mixer/cmd/mixs:mixs
-MIXER_BINARY=$(pwd)/bazel-bin/mixer/cmd/mixs/mixs
+# If GOPATH is not set by the env, set it to a sane value
+GOPATH ?= $(shell cd ../../..; pwd)
+export GOPATH
 
-# Get fortio
-go get -u istio.io/fortio
+# Build mixer binary
+make mixs
+MIXER_BINARY=${GOPATH}/bin/mixs
```

Comment:
As of the start of the month make puts stuff under go/out rather than go/bin, so I can't see how this works.  The path under go/out depends on the os and arch so the most flexible option I can think of is:

MIXER_BINARY=$(make where-is-out)/mixs

Reply:
That's a great point, thanks Matt!